### PR TITLE
Fix labeler

### LINKED ISSUE #2439: Allow labeling and commenting to PRs from forks
Applies to labeler and possible to codecov.
See https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/ and especially https://securitylab.github.com/research/github-actions-preventing-pwn-requests

### PR DESCRIPTION
**Pull request by sungam3r**
fixes #2439

### CODE REVIEW

**Comment by sungam3r** [comment]
But I can't see that labeler workflow was triggered 😕

**Comment by sungam3r** [comment]
Maybe workflow should first be merged into default branch.

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -6,7 +6,13 @@
...
 
 name: Labeler
-on: [pull_request]
+on:
+  pull_request_target:
+    types:
+      - opened        # when PR is opened
+      - edited        # when PR is edited
+      - synchronize   # when code is added
+      - reopened      # when a closed PR is reopened
```
Comment (Shane32) [inline .github/workflows/label.yml:15]: Only “opened” , PLEASE. So painful having it keep improperly label PRs after you’ve already fixed it once. The labeler is completely worthless anyway. But if it must exist, please only on opened.

Comment (sungam3r) [reply .github/workflows/label.yml]: done, I would like to try and if it does not work rollback

**Review by Shane32 — APPROVED** [review]

**Pull request merged**